src/pages/documentation/adobe-media-analytics/tabs/migration-guide.md

```diff
@@ -0,0 +1,283 @@
+<Variant platform="android" section="install-dependency" repeat="2"/>
+
+Download extension with dependencies using [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/)
+
+```diff
+  implementation 'com.adobe.marketing.mobile:core:2.+'
+- implementation 'com.adobe.marketing.mobile:identity:2.+'
+- implementation 'com.adobe.marketing.mobile:analytics:2.+'
+- implementation 'com.adobe.marketing.mobile:media:3.+'
++ implementation 'com.adobe.marketing.mobile:edge:2.+'
++ implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
++ implementation 'com.adobe.marketing.mobile:edgemedia:1.+'
```

Comment:
Are we using edge media:2.+?

Reply:
yes

Reply:
Do you need to change the edgemedia:2.+?